### DISCUSSION TITLE
另一个关于开放issue区的请求

### DISCUSSION THREAD

**Discussion by uubulb** [nezhahq/nezhahq.github.io#105] [General]
很抱歉创建重复话题。旧的话题没能解决问题，问题描述也不详细，~~而且 论坛/discussion duplicate 应该是常事~~，因此我开了一个新的话题

现在哪吒问题反馈主要分散这几个地方：
1. 主机论坛
2. 官方 telegram 群组
3. 此 discussion

它们分别的主要问题：
1. 不是由哪吒社区管理，内容杂乱，开发者没有时间（也没有义务）去专门搜索哪吒相关的问题
2. 有自动清除，搜索信息不方便，所有信息混在一起，不适合问题记录；没有问题模板，增加了开发者处理问题的心智负担
3. 主项目没有入口，文档也没有直接提及，可以说较为“隐蔽”；没有真正解决“issue没人处理，导致项目看起来不活跃”的问题；在 Github 已经有了 issue/pr 的情况下，更多人把这个拿来当成避免污染 issue 区的地方；亦或软件是闭源/专有的，不需要 issue 的核心功能

当然像哪吒这样直接面向最终用户的项目，开放了issue区后被低质量的issue淹没是很可能的（甚至是不可避免的），之前关闭issue区估计也是因为这个。但 issue 仍然有其不可替代性，也是 github 作为现代代码托管平台的最大特色之一，再加上上述提到替代方案的缺点，重新开放还是很有必要的

issue 还能利好贡献者，比如在写新功能 / 修复 bug 时与项目维护者先进行交流，根据维护者的要求再去开发，某种程度上降低了 review 的负担

issue 没人处理的问题，只能说类似的项目也都差不多…… 比如 alist 挂了快200个。以下是我感觉一些可行的缓解 issue 压力的操作：
1. 放宽 nezhahq 加入限制，把脚本/文档这些用户很常用但与项目自身代码无关的东西交由社区管理。
2. 加 issue 模板，actions 自动关闭长时间无人回复的 issue。
3. 把 issue 权限给一些长期参与社区建设及对项目有所了解的人

我觉得 1.0 发布之后，就可以将 nezha 主项目和 agent 的 issue 区开放。如果 nezha 还有转移到 nezhahq 的计划，也可以等到那个时候

**Comment by naiba** [comment]
开着discussion是我想要个徽章🤡不然也关了

**Comment by naiba** [comment]
issue可以现在就可以打开试一下，nezha转移的问题现在还不是时候都没什么人参与，转移了没人更新没人review就烂尾了

**Comment by cantoblanco** [comment]
我赞同试一下 可以再关

**Comment by naiba** [comment]
可以搞个社区管理的一个规范，加入方式，轮值 review，等等

**Comment by uubulb** [comment]
给我个triage吧，我帮忙处理下

**Comment by naiba** [comment]
没懂

**Reply by uubulb** [reply]
github项目设置，邀请collaborator那里有个权限等级叫triage，能处理issue/pr
